From d624c9ecda675dc73447c2e4e95a6330b2999d22 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 16 Apr 2019 18:39:53 +0300
Subject: [PATCH 151/386] arm64: dts: cn9131: move network nodes to board level
 dts

This patch is preparation to SetupB creation.

Change-Id: I73b3dbb6e5eb0c1e21c058e6a92e9acce1195186
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/7723
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Grzegorz Jaszczyk <Grzegorz.Jaszczyk@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9131-db-A.dts | 14 +++++++++++++
 arch/arm64/boot/dts/marvell/cn9131-db.dtsi  | 23 ---------------------
 2 files changed, 14 insertions(+), 23 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9131-db-A.dts b/arch/arm64/boot/dts/marvell/cn9131-db-A.dts
index e0982fb338ea..5dbfaa37e6b3 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db-A.dts
+++ b/arch/arm64/boot/dts/marvell/cn9131-db-A.dts
@@ -13,3 +13,17 @@
 	compatible = "marvell,cn9131-db-A", "marvell,armada-ap807-quad",
 		     "marvell,armada-ap807";
 };
+
+&cp1_ethernet {
+	status = "okay";
+};
+
+/* CON50 */
+&cp1_eth0 {
+	status = "okay";
+	phy-mode = "10gbase-kr";
+	/* Generic PHY, providing serdes lanes */
+	phys = <&cp1_comphy4 0>;
+	managed = "in-band-status";
+	sfp = <&cp1_sfp_eth1>;
+};
diff --git a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
index 9bb5abd29b24..f4636f78e946 100644
--- a/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9131-db.dtsi
@@ -65,29 +65,6 @@
 	status = "okay";
 };
 
-&cp1_ethernet {
-	status = "okay";
-};
-
-/* CON50 */
-&cp1_eth0 {
-	status = "okay";
-	phy-mode = "10gbase-kr";
-	/* Generic PHY, providing serdes lanes */
-	phys = <&cp1_comphy4 0>;
-	managed = "in-band-status";
-	sfp = <&cp1_sfp_eth1>;
-};
-
-/* RJ45 CON55 via 3310 */
-&cp1_eth1 {
-	status = "disabled";
-	phy-mode = "10gbase-kr";
-	phy = <&phy8>;
-	/* TODO: regarding to the spec comphy2 can be used with port 0 only */
-	phys = <&cp1_comphy2 1>;
-};
-
 &cp1_gpio1 {
 	status = "okay";
 };
-- 
2.17.1

